### PR TITLE
initial - mqtt-export service

### PR DESCRIPTION
**Pull request by mteodor**
Signed-off-by: Mirko Teodorovic <mirko.teodorovic@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Mainflux
+# SPDX-License-Identifier: Apache-2.0
+
+BUILD_DIR = build
+SERVICES = users things http ws coap lora influxdb-writer influxdb-reader mongodb-writer \
```
Comment (drasko) [inline Makefile]: This Makefile should be changed.

You should use generic lightweight Makefile from `mqtt-bench` tool.

File: cmd/exporter/main.go [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+	defMqttCert       = "thing.cert"
+	defMqttPrivKey    = "thing.key"
+	defConfPath       = "config.toml"
+	defChanCfgPath    = "/config/channels.toml"
+
+	envNatsURL  = "MF_NATS_URL"
+	envLogLevel = "MF_EXPORTER_LOG_LEVEL"
+	envPort     = "MF_EXPORTER_PORT"
+
+	envMqttHost       = "MF_EXPORTER_MQTT_HOST"
```
Comment (drasko) [inline cmd/exporter/main.go]: Why `EXPORTER`? Please use name of the service - `EXPORT`

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+WORKDIR /go/src/github.com/mainflux/mainflux
+COPY . .
+RUN apk update \
+    && apk add make\
+    && make $SVC \
+    && mv build/mainflux-$SVC /exe
+
+FROM scratch
+# Certificates are needed so that mailing util can work.
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```
Comment (drasko) [inline docker/Dockerfile]: Why is this needed? It should be volume-mounted.

File: go.mod [hunk 4]
```diff
@@ -0,0 +1,18 @@
+module github.com/mainflux/export
+
+go 1.13
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/cisco/senml v0.0.0-20191010185858-4d43ea8da0ed
```
Comment (drasko) [inline go.mod]: Please use our new SenML lib.

File: writers/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Writers
```
Comment (drasko) [inline writers/README.md]: writers?

File: docker/Dockerfile (hunk 3 above)
Comment (mteodor) [reply docker/Dockerfile]: we have the same on core
mounted from where ?

Comment (drasko) [reply docker/Dockerfile]: Use this generic Makefile: https://github.com/mainflux/mainflux/blob/master/tools/mqtt-bench/Makefile

It is the same over here: https://github.com/mainflux/mainflux/blob/master/tools/provision/Makefile

It's simple and reusable.

Comment (mteodor) [reply docker/Dockerfile]: this is dockerfile, dont get it

Comment (drasko) [reply docker/Dockerfile]: No need to copy certificates into the Dockerfile.

Certificates will be broght into container via volumes defined in docker-compose.
